src/internal/m365/graph/errors.go

```diff
@@ -326,8 +328,9 @@ func setLabels(err *clues.Err, msg string) *clues.Err {
 	f := filters.Contains([]string{msg})
 
 	if f.Compare(string(MysiteNotFound)) ||
-		f.Compare(string(MysiteURLNotFound)) {
-		err = err.Label(LabelsMysiteNotFound)
+		f.Compare(string(MysiteURLNotFound)) ||
+		f.Compare(string(FileNotFound)) {
```

Comment:
I'm not comfortable with this change.  Even by graph api standards, calling a drive a file is outrageous.  How sure are we that we:
1. understand the conditions and context of the error.
2. will avoid unwanted collisions with non-drive file-not-found errors.
3. are not seeing a live graph api bug.